assistant_dialog_skill_analysis/utils/skills_util.py

```diff
@@ -156,11 +157,12 @@ def retrieve_workspace(
     :param username:
     :param password:
     :param export_flag:
+    :param disable_ssl_verification:
     :return workspace: workspace json
     """
 
     if iam_apikey:
-        authenticator = IAMAuthenticator(apikey=iam_apikey)
+        authenticator = IAMAuthenticator(apikey=iam_apikey, disable_ssl_verification=disable_ssl_verification)
```

Comment:
Instead of doing it in the authenticator, you can set it later on service too: https://github.com/watson-developer-cloud/python-sdk#disable-ssl-certificate-verification. 

i.e,
```
conversation.set_disable_ssl_verification(disable_ssl_verification)
```

Reply:
make sense to me.